support-frontend/app/views/subscriptionRedemptionForm.scala.html

```diff
@@ -0,0 +1,43 @@
+@import admin.settings.AllSettings
+@import assets.{AssetsResolver, RefPath, StyleContent}
+@import com.gu.identity.model.{User => IdUser}
+@import com.gu.support.encoding.CustomCodecs._
+@import com.gu.support.redemptions.redemptions.RedemptionCode
+@import helper.CSRF
+@import views.ReactDiv
+@import com.gu.support.redemptions.CorporateCustomer
+@import io.circe.syntax._
+
+@import io.circe.Json
+@(
+        title: String,
+        mainElement: ReactDiv,
+        js: String,
+        css: String,
+        fontLoaderBundle: Either[RefPath, StyleContent],
+        csrf: Option[String],
+        uatMode: Boolean,
+        stage: String,
+        redemptionCode: RedemptionCode,
+        customerOrError: Either[String, CorporateCustomer],
+        user: Option[IdUser],
+        submitted: Boolean,
+)(implicit assets: AssetsResolver, requestHeader: RequestHeader, settings: AllSettings)
+
+@main(title = title, mainJsBundle = Left(RefPath(js)), mainElement = mainElement, mainStyleBundle = Left(RefPath(css)), fontLoaderBundle = fontLoaderBundle, csrf = csrf) {
+  <script type="text/javascript">
+      window.guardian = window.guardian || {};
+      window.guardian.stage = "@stage";
+      window.guardian.userCode = "@redemptionCode";
+      window.guardian.error = "@customerOrError.left.getOrElse("")";
+      window.guardian.submitted = @submitted;
+      window.guardian.corporateCustomer = @Html(customerOrError.right.map(_.asJson.noSpaces).getOrElse(Json.Null.noSpaces));
```

Comment:
why the `@Html` and not just @?  adding `Html`  will prevent it being escaped correctly.

Reply:
correction - it will prevent it being escaped INcorrectly, I just tested it! 😆